test/python/linear_attention/test_fla_compat.py

```
@@ -293,6 +293,40 @@ def test_fallback_is_transparent():
+        lv["g"],
+        lv["beta"],
+        output_final_state=True,
+        state_v_first=state_v_first,
+    )
+    got = last_path()
+    if expect_native:
+        assert got == "native", f"state_v_first={state_v_first}: expected native, got {got}"
+        assert fs_cud.shape == (m["q"].shape[0], m["v"].shape[2], m["v"].shape[3], m["q"].shape[3])
+    else:
+        assert got.startswith("fallback"), f"state_v_first={state_v_first}: expected fallback, got {got}"
+        o_fla, fs_fla = chunk_gated_delta_rule(
```

Ruff (0.16.1)

[warning] Unpacked variable `fs_fla` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)